src/functions.js

```diff
@@ -12,45 +18,117 @@ function getNumberOfBikes( document ) {
 };
 
 function getAllButtonText( document ) {
-  // get the text inside all tags with class 'btn' and return as an array of strings
-  
+  var buttonText = document.getElementsByClassName('btn');// get the text inside all tags with class 'btn'
+    var buttonList = [];
+
+    for (i=0; i<buttonText.length; i++){
+     
+      buttonList.push(buttonText[i].innerHTML.trim());
+
+    } // and return as an array of strings
+ return buttonList;
```

Comment:
note on the style (as in, the way you use white spaces, new lines, etc.): you have indented the steps between line 22 and 28 by an extra space, and in the line 29 you removed all of those.

Indentation adds *no* functional differences, but is very useful to help developers visualise the flow of the code. BY having uneven indentation the code is slightly harder to understand and one might be lead in error thinking that the return is outside the function (it's not) and the empty array declaration is inside a loop or if statement (it's not)